### DISCUSSION TITLE
Backups Location

### DISCUSSION THREAD

**Discussion by jaymarvels** [advplyr/audiobookshelf#340] [General]
Is there anyway to change/select where the backups are stored? 

I am running in a container instance and if for some reason the container gets restarted or up/down then any information within the container gets lost, including the backups, but if i can store these on the attached storage then no issues

**Comment by ChiefMedicalOfficer** [comment]
Haven't you setup persistent storage in your docker run command or compose file?

**Comment by jaymarvels** [comment]
I have a mounted file share on my container (azure) which is where I store all my audio books. Where the docker image is attached on the container instance there is no persisted storage, so where Audiobookshelf is installed means that when backups are made its on the container instance and not the mounted storage

**Reply by advplyr** [reply]
I think there is an issue thread still open relating to backup location. I'm thinking the best option might be to just make backup locations their own path instead of `/metadata/backups` you would map `/backups`. This would require some backwards compatibility.
Would that be a viable solution for you?

**Reply by jaymarvels** [reply]
Yep, being able to select the path, similar to how we select the library path would be perfect

**Reply by darnir** [reply]
I would love for this feature to be implemented. I currently have /metadata mounted as a volume as when I initially set up ABS, this didn't look like a directory that I needed direct access to.
Now that I'm trying to set up backups, I see that it can only go to /metadata/backups which is not ideal for me. Please make it accessible as a separate path that I can point it to.

**Reply by ExpatTX** [reply]
I would love this feature to be added as well. In the meantime is there anywhere we can go to edit the location either in a config file or a SQL update?

**Reply by instantdreams** [reply]
I am also voting for the use of `/backups` as the target for backups. Then, we can manage the mapping in docker as needed.
